Gordon360/Controllers/ProfilesController.cs

```diff
@@ -466,6 +485,36 @@ public async Task<ActionResult<FacultyStaffProfileViewModel>> UpdateOfficeHours(
         return Ok(result);
     }
 
+    /// <summary>
+    /// Set visibility of some piece of personal data for user.
+    /// </summary>
+    /// <param name="userPrivacy">Faculty Staff Privacy Decisions (see UserPrivacyUpdateViewModel)</param>
+    /// <returns></returns>
+    [HttpPut]
+    [Route("user_privacy")]
```

Comment:
```suggestion
    [Route("{username}/user_privacy")]
```
This may be personal preference, but I think it's better to always refer to a unique resource identifier rather than assuming the current authenticated user -- especially when trying to follow rest architecture.

Another option would be `[Route("me/user_privacy")]` but I don't know that we use that elsewhere.